Add non-secure configuration for Thingy53 samples (#5740)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

    See the documenation page of nRF Desktop's :ref:`nrf_desktop_hid_forward` for details.
  * Fixed an issue that was causing the HID keyboard LEDs to remain turned on after host disconnection while no other hosts were connected.

Bluetooth® LE
-------------

* Updated:

>   * Updated samples with support for the :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.

Reviewer: ```suggestion
  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
```

>   * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.

Reviewer 2: * Updated ~~some~~ samples with support for...

Reviewer: That "some" was actually added by me. From what I could tell, not all Bluetooth LE samples were updated.

Reviewer 2: Well... it is just a comment. I really do not like "some" in a documentation and the rest of the comment explains with samples configuration was added. I am not blocking merge anyway - As I marked in my approval - if documentation team agrees that I have nothing to add.

Bluetooth mesh
--------------

* Updated:

>   * Updated samples with support for the :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.

Reviewer: ```suggestion
  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
```

>   * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.

Reviewer 2: * Updated ~~some~~ samples with support for...

Author: I added `some` on @wiba-nordic  request :(

Reviewer 2: Just a comment - not a blocker as I stated in previous comment.

Reviewer: I'll ask the rest of the doc team for opinions.

Edge Impulse
------------

> * Added non-secure configuration for build with the :ref:`zephyr:thingy53_nrf5340`.

Reviewer: ```suggestion
* Added non-secure configuration for building with :ref:`zephyr:thingy53_nrf5340`.
```
